spyder/config/main.py

```diff
@@ -288,8 +288,9 @@
               'search_text_regexp': False,
               'search_text': [''],
               'search_text_samples': [TASKS_PATTERN],
-              'more_options': True,
+              'more_options': False,

```

Comment:
You forgot to bump CONF_VERSION to account for this change.

Reply:
🤦‍♂

Reply:
Fixed